flex-config/ui_attributes.common.json

```diff
@@ -263,7 +263,72 @@
           "location": false,
           "agent_skills": true
         }
+      },
+      "queues_view_data_tiles": {
+        "enabled": true,
+        "activeTasksDataTile": false,
+        "waitingTasksDataTile": false,
+        "longestWaitTimeDataTile": false,
+        "agentsByActivityBarChart": false,
+        "channels": {
+          "voice": {
+            "color": "#ADD8E6",
+            "SLADataTile": true,
+            "taskCountsDataTile": true
+          },
+          "chat": {
+            "color": "#87CEFA",
+            "SLADataTile": false,
+            "taskCountsDataTile": false
+          },
+          "sms": {
+            "color": "#4682B4",
+            "SLADataTile": true,
+            "taskCountsDataTile": true
+          },
+          "video": {
+            "color": "#00CED1",
+            "SLADataTile": false,
+            "taskCountsDataTile": false
+          }
+        },
+        "allChannelsDataTile": true,
+        "enhancedAgentByActivityPieChart": true,
+        "agentActivityConfiguration": {
+          "Available": {
+            "color": "green",
+            "icon": "Accept"
+          },
+          "Outbound": {
+            "color": "greenyellow",
+            "icon": "Call"
+          },
+          "Break": {
+            "color": "goldenrod",
+            "icon": "Hold"
+          },
+          "Lunch": {
+            "color": "darkorange",
+            "icon": "Hamburger"
+          },
+          "Training": {
+            "color": "red",
+            "icon": "Bulb"
+          },
+          "OTHER": {
+            "color": "darkred",
+            "icon": "More"
+          },
```

Comment:
Is OTHER a special value that is keyed off later on? If so I think it should be separated out in the config to prevent someone from breaking things.

Reply:
Done. I separated the list of activities from the OTHER category